BabyPatterns/DispatchVC.swift

```diff
@@ -35,6 +49,13 @@ class DispatchVC: UIViewController, Loggable {
             userLoggedIn(user: user)
         } else {
             presentSignup()
+
+            let context = ["loggedOut": "dummy"]
+            do {
+                try WCSession.default.updateApplicationContext(context)
+            } catch(let e) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/d494445f52cbac6dfdfd0acc9da500a5ab222714/BabyPatterns/DispatchVC.swift#L56"></span><code>if</code>, <code>for</code>, <code>guard</code>, <code>switch</code>, <code>while</code>, and <code>catch</code> statements shouldn’t unnecessarily wrap their conditionals or arguments in parentheses.<br />
<code>control_statement</code> <code>DispatchVC.swift:56</code></td>
    </tr>
  </tbody>
</table>